README.md

```diff
@@ -330,7 +330,148 @@ Extracting files and analyzing VoIP calls are just a couple of advanced ways you
 
 
 ## An Overview/Introduction to TShark
-TShark is the terminal oriented version of Wireshark that can capture and display packets without the need for an interactive user interface. Without having any options set, TShark works in a similar way as a tcpdump. To begin capturing packets with TShark you would use the following syntax within the terminal window: 
+TShark is the terminal oriented version of Wireshark that can capture and display packets without the need for an interactive user interface. It supports the same options as Wireshak and without having any options set, TShark works in a similar way as a tcpdump. For more information on Wireshark consult your local manual page (`man shark`) or the online version at (https://www.wireshark.org/docs/man-pages/tshark.html).
+
+### Help information available from `tshark`
+
+`TShark (Wireshark) 3.3.2 (v3.3.2rc0-16-g1936fef77a42)
+Dump and analyze network traffic.
+See https://www.wireshark.org for more information. 
+
+Usage: tshark [options] ...
+
+Capture interface:
+  -i <interface>, --interface <interface>
```

Comment:
The view is a little wonky, could format this text as code for easier readability.